File: 1065.java
import java.util.*;

public class Main {
    public static void main(String args[]) {
    int a, i;
 int tmp = 0;
    Scanner input = new Scanner(System.in);
 for (i = 0; i < 5; ++i)
 {
  a = input.nextInt();
  if(a < 0){
   a = -a;
  }

  if(a % 2 == 0){
   tmp++;
  }
 }

  System.out.printf("%d valores pares\n", tmp);
    }
}

File: 1065.c
#include <stdio.h>
int main()
{
    int i;
    int c=0;
    int n;
    for(i=1;i<=5;i++)
    {
        scanf("%d", &n);
        if (n%2==0)
        {
            c++;
        }


    }
    printf("%d valores pares\n", c);
    return 0;
}
